src/ProjectService.Data/TaskRepository.cs

```diff
@@ -58,8 +58,18 @@ public bool Edit(DbTask task, JsonPatchDocument<DbTask> taskPatch)
 
         public DbTask Get(Guid taskId)
         {
-            return _provider.Tasks.FirstOrDefault(x => x.Id == taskId) ??
-                   throw new NotFoundException($"Task id '{taskId}' was not found.");
+            DbTask dbTask = _provider.Tasks
+                                .Include(t => t.Project)
+                                .Include(t => t.Author)
+                                .Include(t => t.AssignedUser)
+                                .Include(t => t.Status)
+                                .Include(t => t.Priority)
+                                .Include(t => t.Type)
+                                .Include(t => t.Subtasks)
+                                .FirstOrDefault(x => x.Id == taskId) ??
+                throw new NotFoundException($"Task id '{taskId}' was not found.");
+            
+            return dbTask;
```

Comment:
Bad. Edit task command alredy used this method. And this endpoint doesn't need all this information

Reply:
Get endpoint will be used on project page, there are all necessary includes here. What do you want?

Reply:
Add filter

Reply:
I have added another method because it is not find and we should get all or nothing includes